dev/build/windows/configure_profile.sh

```diff
@@ -14,30 +14,29 @@
 rcfile=~/.bash_profile
 donefile=~/.bash_profile.upated
 
+exec >> $rcfile
```

Comment:
Using exec redirection: technically this is fine, but I think this construct is not so well known, so I don't know if this change is good from a understandability / maintainability point of view. My guess is less than 10% of people with good bash experience understand this.

Reply:
Added a comment with a link to understand `exec >>`.